frontend/src/pages/projects/screens/detail/overview/trainModels/NotebooksCard.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import {
+  Button,
+  ButtonVariant,
+  CardBody,
+  Flex,
+  FlexItem,
+  Text,
+  TextContent,
+} from '@patternfly/react-core';
+import { ArrowRightIcon } from '@patternfly/react-icons';
+import { ProjectDetailsContext } from '~/pages/projects/ProjectDetailsContext';
+import { useAccessReview } from '~/api';
+import { AccessReviewResource } from '~/pages/projects/screens/detail/const';
+import { ProjectObjectType, SectionType } from '~/pages/projects/types';
+import { getNotebookDisplayName, typedEmptyImage } from '~/pages/projects/utils';
+import NotebookRouteLink from '~/pages/projects/notebook/NotebookRouteLink';
+import { NotebookDataState } from '~/pages/projects/notebook/types';
+import MetricsCard from '~/pages/projects/screens/detail/overview/components/MetricsCard';
+import OverviewCard from '~/pages/projects/screens/detail/overview/components/OverviewCard';
+
+const notebookSorter = (a: NotebookDataState, b: NotebookDataState) => {
+  if (a.isRunning !== b.isRunning) {
+    return a.isRunning ? -1 : 1;
+  }
+  if (a.isStarting !== b.isStarting) {
+    return a.isStarting ? -1 : 1;
+  }
+  return getNotebookDisplayName(a.notebook).localeCompare(getNotebookDisplayName(b.notebook));
+};
+
+const NotebooksCard: React.FC = () => {
+  const navigate = useNavigate();
+  const [queryParams, setQueryParams] = useSearchParams();
+  const {
+    currentProject,
+    notebooks: { data: notebooks, loaded, error },
+  } = React.useContext(ProjectDetailsContext);
+  const [allowCreate] = useAccessReview({
+    ...AccessReviewResource,
+    namespace: currentProject.metadata.name,
+  });
+
+  const statistics = React.useMemo(
+    () => [
+      {
+        count: notebooks.filter((n) => n.isStopped).length,
+        text: 'Stopped',
+        onClick: () => {
+          queryParams.set('section', 'workbenches');
+          setQueryParams(queryParams);
+        },
+      },
+      {
+        count: notebooks.filter((n) => n.isStarting).length,
+        text: 'Starting',
+        onClick: () => {
+          queryParams.set('section', 'workbenches');
+          setQueryParams(queryParams);
+        },
+      },
+      {
+        count: notebooks.filter((n) => n.isRunning).length,
+        text: 'Running',
+        onClick: () => {
+          queryParams.set('section', 'workbenches');
+          setQueryParams(queryParams);
+        },
+      },
+    ],
+    [notebooks, queryParams, setQueryParams],
+  );
+
+  const listItems = React.useMemo(() => {
```

Comment:
Whenever I see a `useMemo` with JSX it makes be believe that it would better off as a standalone component instead.
`MetricsCard` wants a list of items but it doesn't seem to do anything list like with it. It renders the list as is.